src/game/ai/bat.c

```diff
@@ -88,3 +91,53 @@ void BatControl(int16_t item_num)
 
     CreatureAnimation(item_num, angle, 0);
 }
+
+void InitialiseBat(int16_t item_num)
+{
+    InitialiseCreature(item_num);
+
+    // Almost all of the bats in the OG levels are embedded in the ceiling.
+    // This will move all bats up to the ceiling of their rooms and down
+    // by the height of their hanging animation.
+    if (T1MConfig.fix_bats_position) {
```

Comment:
I don't think we need to have this check in the config, but can we move the entire calculation below to a separate static function? `FixEmbeddedBatPosition`

Reply:
Fixed. Declared that function in the .c file since it has no use outside it.